cypress/support/validations/performanceValidation.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2024 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const CONSTANTS = require('../constants/constants');
+const UTILS = require('../cypress-support/src/utils');
+
+/**
+ * @module performanceValidation
+ * @function performanceValidation
+ * @description Validates performance metrics based on provided validation parameters.
+ * @param {Object} object - The object to validate
+ * @example
+ * cy.performanceValidation({})
+ */
+Cypress.Commands.add('performanceValidation', (object) => {
```

Comment:
I am little confused here.. Is this glue to start performance metrics or validate?

Reply:
This function is to validate performance metrics.